tools/cli/internal/openapi/file.go

```diff
@@ -31,20 +32,49 @@ const (
 	DotJSON = ".json"
 )
 
-func SaveSpec(path string, oas *Spec, format string, fs afero.Fs) error {
-	data, err := NewArrayBytesFromOAS(oas, path, format)
+// SaveToFile saves the content to a file in the specified format.
+// If format is empty, it saves the content in both JSON and YAML formats.
+// Note:
+// - SaveToFile expects the path to be a file path, not a directory path.
```

Comment:
[nit] if that's so, can we make sure to check if path is a directory and throw an error if it is?

Reply:
no, this comment is not needed. I am actually not sure what I meant here 🤦